src/client/marble-manager.js

```diff
@@ -20,7 +22,18 @@ let marbleManager = function() {
 			renderCore.mainScene.add(this.marbleGroup);
 			renderCore.mainScene.add(this.marbleNamesGroup);
 
+			// Default marble model
 			this.marbleGeometry = new THREE.SphereBufferGeometry(1, 32, 32);
+
+			// Default marble texture
+			this.marbleTexture = new THREE.TextureLoader().load(
```

Comment:
From what I can tell this never is used, since the "default" skin is loaded the same way as the other ones. Maybe make this a simple fallback texture that doesn't require loading files, and in `getSkin()` set `this.skins[id]` to that texture in case the texture fails to load?